src/Microsoft.AspNet.Server.Kestrel/Http/Frame.cs

```diff
@@ -71,6 +72,7 @@ public Frame(ConnectionContext context)
             _remoteEndPoint = remoteEndPoint;
             _localEndPoint = localEndPoint;
             _prepareRequest = prepareRequest;
+            _pathBase = context?.ServerAddress?.PathBase;
```

Comment:
I'm fairly certain (maybe with the exception of some tests) that neither context nor ServerAddress can be null.